dev/com.ibm.ws.microprofile.metrics.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,4 +18,8 @@ mpMetrics=MicroProfile Metrics
 mpMetrics.desc=Configuration for MicroProfile Metrics.
 
 authentication=Enable authentication 
-authentication.desc=Indicates whether or not users should provide user name and password when accessing metrics endpoint.
\ No newline at end of file
+authentication.desc=Indicates whether or not users should provide user name and password when accessing metrics endpoint.
```

Comment:
```suggestion
authentication.desc=Indicates whether users must provide a username and password when they access the metrics endpoint.
```

Reply:
no worries, I just reviewed it because it came up in our queue. If Libby tags a PR for ID review unnecessarily, you can skip it by adding the `ID review not needed` label.